Makefile

```diff
@@ -28,7 +28,7 @@ ifeq ($(TARGET_GOARCH), amd64)
 	RACE = -race
 endif
 
-GOFLAGS ?= -a -mod=vendor $(RACE)
+GOFLAGS ?= -a -mod=vendor -buildvcs=false $(RACE)
```

Comment:
I have added this option otherwise, the go compiler complains about missing VCS info: https://stackoverflow.com/questions/76997566/error-obtaining-vcs-status-exit-status-128-use-buildvcs-false-to-disable-vcs-s

Reply:
Is this go compiler error happening on Konflux, or OpenShift CI?

Note that the linked question on Stackoverflow does not have an accepted answer.

Reply:
It is happening in Konflux.

Reply:
This is not happening in OpenShift CI, only in Konflux. OpenShift CI is using a different builder image, maybe it because of image we are using Konflux.